cip/cip30/src/main/java/com/bloxbean/cardano/client/cip/cip30/CIP30ValueSerializer.java

```diff
@@ -0,0 +1,50 @@
+package com.bloxbean.cardano.client.cip.cip30;
+
+import co.nstant.in.cbor.model.Array;
+import co.nstant.in.cbor.model.DataItem;
+import co.nstant.in.cbor.model.UnsignedInteger;
+import com.bloxbean.cardano.client.exception.CborSerializationException;
+import com.bloxbean.cardano.client.transaction.spec.Value;
+import com.bloxbean.cardano.client.transaction.util.CborSerializationUtil;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Serialize Value Object to cbor CIP30's Balance Object
+ * to Support CIP30's wallet.getUtxo(amount: cbor&lt;value&gt;) method
+ */
+public class CIP30ValueSerializer {
```

Comment:
Same as CIP30Deserializer, do we need this class ? We can keep this class but it will invoke Value class.

Reply:
We don't need these Classes as long as the serialization & deserialization functions have the same functionality as in these classes.
I agree that we should have them in one single place. that would be even better.